MatterControlLib/Library/Export/FolderExport.cs

```diff
@@ -89,10 +89,10 @@ public async Task<bool> Generate(IEnumerable<ILibraryItem> libraryItems, string
 					}
 				});
 
-				return true;
+				return null;
```

Comment:
Any reason to return null rather than an empty enumerable?

```suggestion
				return Enumerable.Empty<List<string>>();
```

Reply:
I think this is a better approach. It makes it clear that we test for content rather than null.